### ISSUE TITLE
After updating hosts Internet does not work -> Disable DNS Service needs error management

### ISSUE THREAD

**Issue by mrkaban** [LV-Crew/Adblocker#1]
Hello, after the hosts file is updated, websites no longer open. I tried both black lists, and even without disabling the DNS service, on two different computers (Windows 10 + Windows 7). A very interesting program and pleases the license gnu gpl, but all sites cease to open.

**Comment by Tobias-B-Besemer** [comment]
Dear @mrkaban,

have you solved the issue ???

That some pages no more open after you installed the list is normal - that's what the app should do. ;-)
But e.g. google.com should work...
You should disable your Windows DNS when you use a big list like that from hosts-file.net...
...but at least after a reboot of the system, all should work fine.

If you still have problems to get the adblocker running, please feel free to add a comment...
...if now all works fine, please comment and/or close the issue.
Thanks in advance!

Greets, Tobias.

Btw.: Thank you for 'A very interesting program and pleases the license gnu gpl'! :-)

**Comment by mrkaban** [comment]
It hangs on disabling DNS, after restarting, even Google does not open. The Internet starts working after cleaning up the hosts file.
![default](https://user-images.githubusercontent.com/14289360/36069484-5c97aa8c-0f1d-11e8-937e-257ac9df4a00.jpg)

**Comment by Tobias-B-Besemer** [comment]
Dear @mrkaban,

thank you for your feedback!

If you wanna use the big hosts-file.net-list, you need to disable the DNS Service.
Seems - why ever - your system hangs at this point.

Please try the following:
Try to disable the DNS Service first from the Tools-Page - there's a button too for this.
If this fails, please try to disable the DNS Service manual via 'Services' of your OS.
When the DNS Service is shown as disabled (manual or on the Tools-Page), you can try to install the list again.
If the DNS Service hangs e.g. by disabling him, you maybe need to reboot your system first.

Please give me a feedback if it works with a disabled, stopped DNS Service (after reboot).
Thank you in advance!

Greets, Tobias.

**Comment by mrkaban** [comment]
Yes, it worked. However, the program still hangs on disabling the DNS client, and you have to force reboot the computer, and after rebooting manually disable the DNS client. Also there is a delay when opening web pages, very large.

**Comment by Tobias-B-Besemer** [comment]
@mrkaban, you can try the following:
Set the option to redirect to 0.0.0.0 and download the list(s) again.
Then there is no timeout from trying to find our blockpage-server that is currently down.
That should speed up your loading-performance...
Feedback would be nice, so we can improve this behavior in following versions.

**Comment by mrkaban** [comment]
I added a program to my site about free software. Described there the solution of the problem, about which you wrote here, since I have it on three computers with different versions of Windows. Here is the [link](https://xn--90abhbolvbbfgb9aje4m.xn--p1ai/%D0%B1%D0%B5%D0%B7%D0%BE%D0%BF%D0%B0%D1%81%D0%BD%D0%BE%D1%81%D1%82%D1%8C/%D0%BF%D1%80%D0%BE%D1%87%D0%B5%D0%B5/lv-crew-adblocker-%D0%B0%D0%BD%D1%82%D0%B8%D0%B1%D0%B0%D0%BD%D0%BD%D0%B5%D1%80.html).

I ask you, please try to solve the problem described above, since it occurs on three computers (Windows 7x64 and Windows 10 x64). It makes work very difficult and I'm afraid that ordinary users will not be able to figure it out.

**Comment by Tobias-B-Besemer** [comment]
Dear @mrkaban,

thank you very much for mention the program!

Thanks you, we now aware of the problem!
But unfortunately I don't know till when it will be fixed...

Greets, Tobias.

Btw.: This program is 'just a clone' of our program that you can find here:
https://github.com/LV-Crew/HostsManager

**Comment by mrkaban** [comment]
and the program "Adblocker" will develop? just as an ad blocker it is more interesting!

**Comment by Tobias-B-Besemer** [comment]
In fact just the name differ... ;-)

The problem at the moment is, that Dennis, the coder, is no more interested in future development... :-/
...so I search for a new .NET-Developer... :-/

**Comment by mrkaban** [comment]
I'm only in the development environment Lazarus can write))))) I ask if you will develop the program, under this name, be sure to develop.

The fact is that the hosts have enough editors and the name does not speak of its advantage, and there is simply no free anti-banner.

**Comment by Tobias-B-Besemer** [comment]
If I find a coder that want to participate, the development of this program will continue. ;-)
